src/hera/workflows/v5/examples/coinflip.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: argoproj.io/v1alpha1
+kind: Workflow
+metadata:
+  annotations:
+    workflows.argoproj.io/description: This is an example of coin flip defined as
+      a sequence of conditional steps.
+  generateName: coinflip-
+  namespace: default
```

Comment:
`namespace` isn't used in upstream examples, I see an empty namespace is equivalent to "default" but I don't think we need to include it here - I see it's because in the `GlobalConfig`, `namespace` is set to `"default"`, which is then set in the `Workflow` class as a default value [here](https://github.com/samj1912/hera-workflows/blob/91385cdda1c75fe4bfaeb889caf2306a7c386835/src/hera/workflows/v5/workflow.py#L72).

Reply:
the yaml is auto-generated from the python code, this happens because of the default global config.